lib.sh

```diff
@@ -715,7 +715,13 @@ cd $HTML
 print_text_in_color "$ICyan" "Downloading $STABLEVERSION..."
 curl -fSLO --retry 3 "$NCREPO/$STABLEVERSION.tar.bz2"
 mkdir -p "$GPGDIR"
-curl_to_dir "$NCREPO" "$STABLEVERSION.tar.bz2.asc" "$GPGDIR"
+if network_ok
+then
+    curl_to_dir "$NCREPO" "$STABLEVERSION.tar.bz2.asc" "$GPGDIR"
+else
+    msg_box There seems to be an issue with your network, please try again later.\nThis script will exit."
```

Comment:
I think this should be:
`    msg_box "There seems to be an issue with your network, please try again later.\nThis script will exit."`

Reply:
```suggestion
    msg_box There seems to be an issue with your network, please try again later.\nThis script will exit."
```

Yes, good catch!